src/azure-cli/azure/cli/command_modules/cdn/_params.py

```diff
@@ -262,3 +221,305 @@ def load_arguments(self, _):
         c.argument('request_threshold', type=int)
         c.argument('duration', type=int)
+
+    # AFDX
+    with self.argument_context('afd') as c:
+        c.argument('tags', tags_type)
+
+    # AFD endpoint #
+    with self.argument_context('afd endpoint') as c:
+        c.argument('origin_response_timeout_seconds', type=int,
+                   options_list=["--origin-response-timeout-seconds"],
+                   help="Send and receive timeout on forwarding request to the origin. "
+                        "When timeout is reached, the request fails and returns.")
+        c.argument('profile_name', help=profile_name_help, id_part='name')
+        c.argument('enabled_state', arg_type=get_enum_type([item.value for item in list(EnabledState)]),
+                   options_list=["--enabled-state"],
+                   help="Whether to enable this endpoint.")
+        c.argument('endpoint_name', endpoint_name_type, id_part='child_name_1',
+                   help='Name of the endpoint under the profile which is unique globally.')
+        c.argument('location', validator=get_default_location_from_resource_group)
+        c.argument('content_paths', nargs='+',
+                   help="The path to the content to be purged. Can describe a file path or a wildcard directory.")
+        c.argument('domains', nargs='+', help='List of domains.')
+
+    with self.argument_context('afd endpoint list') as c:
+        c.argument('profile_name', id_part=None)
+
+    # AFD Origin Group #
+    with self.argument_context('afd origin-group') as c:
+        c.argument('origin_group_name', origin_group_name_type, id_part='child_name_1',
+                   help='Name of the origin group.')
+        c.argument('profile_name', help=profile_name_help, id_part='name')
+
+        # health probe related paramters
+        c.argument('probe_request_type', arg_type=get_enum_type([item.value for item in list(HealthProbeRequestType)]),
+                   help='The type of health probe request that is made.')
+        c.argument('probe_protocol', arg_type=get_enum_type([item.value for item in list(ProbeProtocol)]),
+                   help='Protocol to use for health probe.')
+        c.argument('probe_interval_in_seconds', type=int,
+                   help='The number of seconds between health probes.Default is 240 seconds.')
+        c.argument('probe_path',
+                   help="The path relative to the origin that is used to determine the health of the origin.")
+
+        # load balancing related parameters
+        c.argument('load_balancing_sample_size', type=int,
+                   help="The number of samples to consider for load balancing decisions.")
+        c.argument('load_balancing_successful_samples_required', type=int,
+                   help="The number of samples within the sample period that must succeed.")
+        c.argument('load_balancing_additional_latency_in_milliseconds', type=int,
```

Comment:
please add a short name as the alias in options_list property

Reply:
I've trip the leading "load_balancing" in the options_list, but it is still a little longer than the limit.

I could not find a suitable short name for those parameters, do you have any suggestion for that? :)